<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/logo/logo_24x24.png</normaloff>:/icons/logo/logo_24x24.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="MainWidget" name="mainWidget" native="true"/>
      <widget class="SpliceGraphicsWidget" name="spliceGraphicsWidget" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_AboutQt"/>
   </widget>
   <widget class="QMenu" name="menuE_xamples">
    <property name="title">
     <string>E&amp;xamples</string>
    </property>
    <addaction name="action_4BoltJoint"/>
    <addaction name="action_PatternJoint"/>
    <addaction name="action_RandomPattern"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Add"/>
    <addaction name="action_Duplicate"/>
    <addaction name="action_Remove"/>
    <addaction name="separator"/>
    <addaction name="action_SelectAll"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuE_xamples"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="action_SaveAs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="SpliceToolbar" name="spliceToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/file_new_24x24.png</normaloff>:/icons/menu/file_new_24x24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/file_open_24x24.png</normaloff>:/icons/menu/file_open_24x24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/file_save_24x24.png</normaloff>:/icons/menu/file_save_24x24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/file_save_as_24x24.png</normaloff>:/icons/menu/file_save_as_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_4BoltJoint">
   <property name="text">
    <string>4-Bolt Joint</string>
   </property>
  </action>
  <action name="action_PatternJoint">
   <property name="text">
    <string>Pattern Joint</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>About &amp;FastenerPattern...</string>
   </property>
  </action>
  <action name="action_RandomPattern">
   <property name="text">
    <string>Random Pattern</string>
   </property>
  </action>
  <action name="action_AboutQt">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
  <action name="action_Add">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/bolt_add_24x24.png</normaloff>:/icons/menu/bolt_add_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Add new fastener</string>
   </property>
  </action>
  <action name="action_Remove">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/bolt_delete_24x24.png</normaloff>:/icons/menu/bolt_delete_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="action_SelectAll">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/bolt_select_all_24x24.png</normaloff>:/icons/menu/bolt_select_all_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="action_Duplicate">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/bolt_duplicate_24x24.png</normaloff>:/icons/menu/bolt_duplicate_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Duplicate</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SpliceGraphicsWidget</class>
   <extends>QWidget</extends>
   <header>Editor/SpliceGraphicsWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpliceToolbar</class>
   <extends>QToolBar</extends>
   <header>Widgets/SpliceToolbar</header>
  </customwidget>
  <customwidget>
   <class>MainWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/MainWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
